client/packages/common/src/ui/components/navigation/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -41,7 +41,12 @@ export const Breadcrumbs = ({
 
   const crumbs = urlParts.map((part, index) => {
     if (index === urlParts.length - 1) {
-      return <span key={part.key}>{suffix ?? parseTitle(part)}</span>;
+      if (!suffix) return <span key={part.key}>{parseTitle(part)}</span>;
+      return typeof suffix === 'string' ? (
+        <span key={part.key}>{suffix}</span>
+      ) : (
+        suffix
+      );
```

Comment:
is this easier to read?
```suggestion
      switch (true) {
        case !suffix:
          return <span key={part.key}>{parseTitle(part)}</span>;
        case typeof suffix === 'string':
          return <span key={part.key}>{suffix}</span>;
        default:
          return suffix;
      }
```

Reply:
Oh interesting haven't though about using a switch statement like this :) Cool and looks "modern" but I would go with the old and boring solution (IMHO a little bit more straight forward):

```
if (!suffix) {
    return <span key={part.key}>{parseTitle(part)}</span>;
}
if (typeof suffix === 'string') {
    return <span key={part.key}>{suffix}</span>;
}
return suffix;
```

Reply:
Well, based off those conflicting suggestions -- I think a simple additional empty line makes it clear enough 😆

```ts
if (!suffix) return <span key={part.key}>{parseTitle(part)}</span>;

return typeof suffix === 'string' ? (
  <span key={part.key}>{suffix}</span>
) : (
  suffix
);
```

Reply:
> Oh interesting haven't though about using a switch statement like this :) 

I picked it up from @CarlosNZ 😉 I'm trying to make a `match` in javascript, I think!